ACD-315 (#5)
Actindo-AG/Documentation: PIM/Integration/06_ManageCatalogs.md

[!!User Interface Catalogs](../UserInterface/03f_Catalogs.md)
[!!Manage the products](../Operation/01_ManageProducts.md)

# Manage a catalog

You can manage products in different catalogs and assign them to categories in order to sort the products and find them faster.

[comment]: <> (UI will be changed)

## Create a catalog

Create a catalog with at least one category for new attributes you want to assign to a certain category.

### Define the catalog basic data

Create a new catalog and define its name to be able to create one or more categories afterwards.

#### Prerequisites

No prerequisites to fulfill.

#### Procedure
*PIM > Settings > Tab CATALOGS*

![Catalogs](../../Assets/Screenshots/PIM/Settings/Catalogs/Catalogs.png "[Catalogs]")

1. Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the catalog name in the left column.   

>   The menu is displayed.

Reviewer: ```suggestion
  The context menu is displayed.
```

  ![Menu](../../Assets/Screenshots/PIM/Settings/Catalogs/Menu.png "[Menu]")

2. Click the *+ Create new* menu entry.   
  The *Create new catalog* view is displayed.

  ![Create new catalog](../../Assets/Screenshots/PIM/Settings/Catalogs/CreateNewCatalog.png "[Create new catalog]")

3. Enter a name for the catalog in the text field in the header.    

> 4. Continue to [add a category](#create-a-category) to the catalog, as at least one category per catalog is required to which you can assign the products.

Reviewer: ```suggestion
4. At least one category per catalog is required to which you can assign the products. Follow the procedure below to add a category, see [Add a category](#create-a-category). 
```
(Oder sowas / evtl. vereinheitlichen, Add oder create?)

> ### Create a category

Reviewer: Add vs. create (vgl. oben)

You need to add at least one category to a catalog to be able to assign products to it.

#### Prerequisites

> The catalog basic data has been created, see [Define the catalog basic data](#define-the catalog-basic-data).

Reviewer: ```suggestion
The catalog basic data has been created, see [Define the catalog basic data](#define-the-catalog-basic-data).
```

#### Procedure
*PIM > Settings > Tab CATALOGS > Button Create new*

1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
  A plus sign is displayed in the category column.

  ![Add category](../../Assets/Screenshots/PIM/Settings/Catalogs/AddCategory.png "[Add category]")

2. Click the plus sign in the category column.    
  The *Add Element* window is displayed in the category column.

  ![Add element](../../Assets/Screenshots/PIM/Settings/Catalogs/AddElementTop.png "[Add element]")

3. Enter a name for the category of the catalog in the *Name* field.

4. Enter a key for the category of the catalog in the *Key* field.

5. Click the [SAVE] button.   
  The catalog category has been created. The *Add Element* window is closed. The new category is displayed in the category column.

> 

Reviewer: Wenn ich versuche, ein Catalog zu speichern (mit oder ohne category), bekomme die Fehlermeldung: Fields "key" could not be validated. Was mache ich falsch? Was muss man bei Key beachten? Evtl. Info dazu hinzufügen?

  ![Category created](../../Assets/Screenshots/PIM/Settings/Catalogs/CategoryCreated.png "[Category created]")

  > [Info] To add an additional category, repeat the steps **1** to **5**. To add a sub-category, see [Create a sub-category](#create-a-sub-category).  

6. Click the [SAVE] button in the upper right corner.  
  The new catalog has been saved. The *Create new catalog* view is closed.

## Edit a catalog

After you have created a catalog, you can edit the catalog and its categories. You can create further categories, change the order of categories and delete categories.

### Create a sub-category

Create a sub-category to be able to assign products to individual categories in an even more detailed way.

#### Prerequisites

At least one catalog with one category has been created, see [Create a catalog](#create-a-catalog).

#### Procedure
*PIM > Settings > Tab CATALOGS*

![Catalogs](../../Assets/Screenshots/PIM/Settings/Catalogs/Catalogs.png "[Catalogs]")

> 1. Select the catalog you want to create a sub-category to in the drop-down list in the left column.   

Reviewer: ```suggestion
1. Select the catalog where you want to create a sub-category in the drop-down list in the left column.   
```
Unsicher über den Ausdruck "create a category to..." (=in welchem Katalog). Where?

  The catalog name is displayed in the drop-down list.

2. Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the selected catalog.   

>   The menu is displayed.

Reviewer: ```suggestion
  The context menu is displayed.
```

  The *Edit catalog* view is displayed.

  ![Edit Catalog](../../Assets/Screenshots/PIM/Settings/Catalogs/EditCatalog.png "[Edit Catalog]")

4. Select the category to which you want to create a sub-category in the category column.
  A sub-category column is displayed to the right of the category column.

5. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
  A plus sign is displayed in the category column as well as in the sub-category column.

  ![Add sub category](../../Assets/Screenshots/PIM/Settings/Catalogs/AddSubCategory.png "[Add sub category]")

6. Click the plus sign in the sub-category column.    
  The *Add Element* window is displayed in the sub-category column.

  ![Add sub element](../../Assets/Screenshots/PIM/Settings/Catalogs/AddSubElement.png "[Add sub element]")

7. Enter a name for the sub-category in the *Name* field.

8. Enter a key for the sub-category in the *Key* field.

9. Click the [SAVE] button.   
  The catalog sub-category has been created. The *Add Element* window is closed. The new sub-category is displayed in the sub-category column.

>   > [Info] To add an additional sub-category, repeat the steps **4** to **8**. You can create an unlimited number of sub-categories in an unlimited number of levels. The procedure to create a sub-category in another level is identical to creating a sub-category. Just select the category or sub-category to which you want to create another sub-category and click the corresponding plus sign in the column of the respective level.

Reviewer: ```suggestion
  > [Info] To add an additional sub-category, repeat the steps **4** to **8**. You can create an unlimited number of sub-categories in an unlimited number of levels. The procedure to create a sub-category in another level is identical to creating a sub-category. Just select the category or sub-category in which / where you want to create another sub-category and click the corresponding plus sign in the column of the respective level.
```
s.o.

[comment]: <> (number of levels depends on screen size -> should it not be limited to a certain number of level?)

10. Click the [SAVE] button in the upper right corner.   
  The new sub-category has been saved. The *Edit catalog* view is closed.

### Change the order of categories

> The order of categories within the catalog is also displayed in the products view. If you want to have a different order, change the order of categories in the desired way.

Reviewer: ```suggestion
The order of categories within the catalog is also displayed in the product view. If you want to have a different order, change the order of categories in the desired way.
```

#### Prerequisites

- At least one catalog has been created, see [Create a catalog](#create-a-catalog).
- At least two categories have been created within the catalog, see [Create a category](#create-a-category).

#### Procedure
*PIM > Settings > Tab CATALOGS*

![Catalogs](../../Assets/Screenshots/PIM/Settings/Catalogs/Catalogs.png "[Catalogs]")

> 1. Select the catalog you want to create a sub-category to in the drop-down list in the left column.   

Reviewer: s.o.

  The catalog name is displayed in the drop-down list.

2. Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the selected catalog.   

>   The menu is displayed.

Reviewer: ```suggestion
  The context menu is displayed.
```

  ![Edit Catalog](../../Assets/Screenshots/PIM/Settings/Catalogs/EditCatalog.png "[Edit Catalog]")

  > [Info] The procedure to change the order of categories and sub-categories is identical. Just select the sub-category instead of the category and follow the steps described below.

4. Click and hold the ![Sort](../../Assets/Icons/Sort01.png "[Sort]") (Sort) button to the left of the category you want to move to another position in the list.

3. By using drag and drop, move the selected category to the desired position in the list.

4. Click the [SAVE] button in the upper right corner.  
  The new category order has been saved. The *Edit category* view is closed.

### Delete a category

Delete a category that is no longer needed. If you delete a category containing one or more sub-categories, these sub-categories will be deleted as well.

#### Prerequisites

- At least one catalog has been created, see [Create a catalog](#create-a-catalog).
- At least one category has been created within the catalog, see [Create a category](#create-a-category).

#### Procedure
*PIM > Settings > Tab CATALOGS*

![Catalogs](../../Assets/Screenshots/PIM/Settings/Catalogs/Catalogs.png "[Catalogs]")

> 1. Select the catalog you want to delete a category from in the drop-down list in the left column.   

Reviewer: vgl. oben (create to / delete from) -> würde Relativ where auch hier passen?

  The catalog name is displayed in the drop-down list.

2. Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the selected catalog.   

>   The menu is displayed.

Reviewer: ```suggestion
  The context menu is displayed.
```
